nameable.rb

```diff
@@ -0,0 +1,8 @@
+# Create a class nameable a method correct_name which raises a NotImplementedError
+# with a message indicating that the correct name must be added
+
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Must implement the correct name'
+  end
```

Comment:
Nice job here :+1: . However, this will give a generic message that will be difficult to debug and know exactly which class and method the error is coming from. So kindly consider updating the message with the class name and method. 
You can do something like: 
`raise NotImplemetedError, "#{self.class} has not implemented method '#{__method__}'"`